beard/metrics/tests/test_text.py

```diff
@@ -10,12 +10,13 @@
 """Test text metrics.
 
 .. codeauthor:: Petros Ioannidis <petros.ioannidis91@gmail.com>
+.. codeauthor:: Evangelos Tzemis <evangelos.tzemis@gmail.com>
 
 """
 from __future__ import generators
 import pytest
 from pytest import mark
-from ..text import find_all, _jaro_matching, jaro, jaro_winkler
+from ..text import find_all, _jaro_matching, jaro, jaro_winkler, levenshtein
```

Comment:
Can reorder these imports, from the standard library to local imports. Could you also make one import per line?